scipy/linalg/tests/test_lapack.py

```diff
@@ -2923,3 +2924,29 @@ def test_ptsvx_NAG(d, e, b, x):
     df, ef, x_ptsvx, rcond, ferr, berr, info = ptsvx(d, e, b)
     # determine ptsvx's solution and x are the same.
     assert_array_almost_equal(x, x_ptsvx)
+
+
+@pytest.mark.parametrize('dtype', DTYPES)
+def test_pptrs_pptrf_ppsv(dtype):
+    seed(1234)
+    # Manual conversion to/from packed format is feasible here.
+    n, nrhs = 10, 4
+    a = generate_random_dtype_array([n, n], dtype=dtype)
+    b = generate_random_dtype_array([n, nrhs], dtype=dtype)
+    a = a.conj().T + a + np.eye(n, dtype=dtype) * dtype(5.)
+    inds = ([x for y in range(1, n+1) for x in range(y)],
+            [y-1 for y in range(1, n+1) for x in range(y)])
+    ap = a[inds]
+    ppsv, pptrf, pptrs = get_lapack_funcs(('ppsv', 'pptrf', 'pptrs'),
+                                          dtype=dtype)
+
+    ul, info = pptrf(n, ap)
+    aul = cholesky(a)[inds]
+    assert_allclose(ul, aul, rtol=np.finfo(dtype).eps*50)
+
+    x, info = pptrs(n, ul, b)
```

Comment:
Shouldn't the routine be able to figure out `n`?

Reply:
See #12333 for a similar discussion

Reply:
By the time I got there, I forgot it was in a packed format.